system/api-server/src/file-upload/file-upload.controller.ts

```diff
@@ -48,7 +48,7 @@ export class ImageUploadController {
         amount: {type: 'string', example: '100'},
```

Comment:
Should this be a decimal since it's an amount? Any type of number validation for currency?

Reply:
Date should likely also not be a string and have validation

Reply:
Actually, no currency validation was added here. I made it a string so that any value could be collected.

Reply:
I'll make changes